### PR TITLE
Added radial inventory menus for turfs, flat structures, and containers.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
image

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Testing with a high number of items: https://cdn.discordapp.com/attachments/314829683780747264/323513220625334272/unknown.png

Still works ok, but I think there's a bug wth my math relating to the bottom left hand corner.

File: code/objects/structures/lectern.dm [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+// Placeholder.
+/obj/structure/lectern/New()
+	if(prob(60))
+		filled = pick(list("scroll","book","paper"))
+	..()
+// End placeholder.
+
+/obj/structure/lectern/UpdateIcon()
+	..()
+	if(filled) overlays += filled
```
Comment (ElArGee) [inline code/objects/structures/lectern.dm]: :squints:

Comment (MistakeNot4892) [reply code/objects/structures/lectern.dm]: Ah dammit I meant to fix that up after the icons were working.

**Comment by MistakeNot4892** [comment]
https://cdn.discordapp.com/attachments/314829683780747264/323719644466905088/unknown.png
Crafting items uses a radial menu now too.

**Comment by MistakeNot4892** [comment]
image
Building turfs too.

**Comment by MistakeNot4892** [comment]
image
And finally, structures. This is ready for review now.

**Comment by ElArGee** [comment]
Middle clicking on crafting blueprints doesn't display a description.

I wonder if we should bring back the bottom bar that displays what you are moused over?

**Comment by MistakeNot4892** [comment]
Fixed the middleclick oversight. If mouseover wasn't so heavy on network traffic I'd suggest having tooltips.

**Pull request merged**